copy/copy.go

```diff
@@ -646,6 +688,27 @@ func (c *copier) copyOneImage(ctx context.Context, policyContext *signature.Poli
 	// If encrypted and decryption keys provided, we should try to decrypt
 	ic.diffIDsAreNeeded = ic.diffIDsAreNeeded || (isEncrypted(src) && ic.c.ociDecryptConfig != nil) || ic.c.ociEncryptConfig != nil
 
+	options.OptimizeDestinationImageAlreadyExists = true // TODO: PR: Remove forced enablement for PR testing
+	// If enabled, fetch and compare the destination's manifest. And as an optimization skip updating the destination iff equal
+	// To be enabled, options.OptimizeDestinationImageAlreadyExists must be true, the image must not be signed or encryted. And must
+	// not have any pending manifest updates
```

Comment:
(I’m not sure that this comment helps much — it just repeats the code, doesn’t explain anything.)